libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -215,7 +215,10 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
     public submitId = '';
     public menuId = '';
     public dir = 'ltr';
+    currentSearchSuggestionAnnoucementMessage = '';
 
+    private _searchSuggestionAnnoucementMessagePart1 = 'suggestions found';
+    private _searchSuggestionAnnoucementMessagePart2 = 'use up and down arrows to navigate';
```

Comment:
What if the user wants to customize this message?

Reply:
added input property to get the message.